ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -54,7 +54,7 @@ public void restoreStreams() {
   @Test
   public void shouldRunCorrectsTest() throws Exception {
     final String testFolderPath = "src/test/resources/test-runner/";
-    for (int i = 1; i <= 3; i++) {
+    for (int i = 1; i <= 4; i++) {
```

Comment:
we should just find all folders that match `test[0-9]+`. Can be done in a follow-up PR but please make sure to do this

Reply:
Yeah, I can have this in a follow up PR. Tracking it here: #2969